source/Halibut.Tests/Support/EnvironmentVariableReaderHelper.cs

```diff
@@ -0,0 +1,19 @@
+﻿using System;
+using Castle.Core.Internal;
+
+namespace Halibut.Tests.Support
+{
+    public class EnvironmentVariableReaderHelper
+    {
+        public static bool EnvironmentVariableAsBool(string envVar, bool defaultValue)
+        {
+            var value = Environment.GetEnvironmentVariable(envVar);
+            if (string.IsNullOrEmpty(value))
```

Comment:
A small thing, but I find string.IsNullOrWhitespace a little better in these cases. Someone shouldn't be setting it to "  "